src/app/teacher/teacher-run-list/teacher-run-list.component.html

```diff
@@ -58,23 +58,59 @@
       </ng-container>
     </p>
   </div>
-
+  <div
+    *ngIf="filteredRuns.length > 0"
+    class="select-all-controls"
+    fxLayout="row"
+    fxLayoutAlign="start center"
+    fxLayoutGap="20px"
+  >
+    <select-runs-controls
+      [numSelectedRuns]="numSelectedRuns"
+      [numTotalRuns]="filteredRuns.length"
+      (selectRunsOptionChosenEvent)="selectRunsOptionChosen($event)"
+    ></select-runs-controls>
+    <div *ngIf="numSelectedRuns > 0" class="num-selected-runs-display" i18n>
+      {{ numSelectedRuns }} Run(s) Selected
+    </div>
+    <button
+      mat-raised-button
+      color="accent"
+      *ngIf="!showArchived && numSelectedRuns > 0"
+      (click)="archiveSelectedRuns()"
+      i18n
+    >
+      Archive Selected
+    </button>
+    <button
+      mat-raised-button
+      color="accent"
+      *ngIf="showArchived && numSelectedRuns > 0"
+      (click)="unarchiveSelectedRuns()"
+      i18n
+    >
+      Unarchive Selected
+    </button>
+  </div>
```

Comment:
TeacherRunListComponent is getting big. Can we make this block into a new component?